index.bs

```diff
@@ -1314,6 +1315,15 @@ The dictionary SHALL have the following attributes:
    [Section 5.3](https://www.rfc-editor.org/rfc/rfc9002#section-5.3).
 : <dfn for="WebTransportStats" dict-member>minRtt</dfn>
 :: The minimum round-trip time observed on the entire connection.
+: <dfn for="WebTransportStats" dict-member>estimatedSendRate</dfn>
+:: The estimated rate at which queued data will be sent by the user agent, in bits per second.
+   This rate applies to all streams and datagrams that share a [=WebTransport session=]
+   and is calculated by the congestion control algorithm (potentially chosen by
+   {{WebTransport/congestionControl}}). If this [=WebTransport session=] is
+   pooled with others in a shared [=connection=], the value is undefined.
+   If the user agent does not have an estimate, the value is undefined.
+   The value change from defined to undefined or undefined to defined at any time.
```

Comment:
For dictionary members, it's best practice to test for [existence](https://infra.spec.whatwg.org/#map-exists) (lack of existence only produces `undefined` in JS). Keeping the current semantics, this would be:
```suggestion
   pooled with others in a shared [=connection=], or if the user agent does not
   currently have an estimate, the member MUST be [=map/exist|absent=].
   The member can be absent even if present in previous results.
```
...or if we go with nullable (which I recommend), we could write:
```suggestion
   pooled with others in a shared [=connection=], the value is `null`.
   If the user agent does not have an estimate, the value is `null`.
   The value can be `null` even if it were non-null in previous results.
```